ui/src/app/common/services/sinks/sinks.service.ts

```diff
@@ -1,122 +1,120 @@
-import { HttpClient, HttpParams } from '@angular/common/http';
-import { Injectable } from '@angular/core';
-import { Observable } from 'rxjs';
+import {HttpClient, HttpParams} from '@angular/common/http';
+import {Injectable} from '@angular/core';
+import {Observable} from 'rxjs';
 import 'rxjs/add/observable/empty';
 
-import { environment } from 'environments/environment';
-import { Sink } from 'app/common/interfaces/sink.interface';
-import { NotificationsService } from 'app/common/services/notifications/notifications.service';
-import { PageFilters } from 'app/common/interfaces/mainflux.interface';
+import {environment} from 'environments/environment';
+import {Sink} from 'app/common/interfaces/sink.interface';
+import {NotificationsService} from 'app/common/services/notifications/notifications.service';
+import {PageFilters} from 'app/common/interfaces/mainflux.interface';
 
+// default filters
 const defLimit: number = 20;
```

Comment:
Constants UPPERCASE

Reply:
This constants are defined everywhere in mainflux codebase following this camelCase style.

I'll try and update these together with the lint fixes.